components/automate-ui/src/app/page-components/services-sidebar/services-sidebar.component.html

```diff
@@ -1,9 +1,11 @@
-<chef-notification
-  *ngIf="(serviceGroupsStatus$ | async) === 'loadingFailure'"
-  [type]="'error'"
-  [timeout]="10">
-  {{ (serviceGroupsError$ | async).status }}: {{ (serviceGroupsError$ | async).error.message }}
-</chef-notification>
+<div class="sticky-notifications">
```

Comment:
Seems like you are using this wrapper a lot... almost feels like that should be a parameter of `<app-chef-notification>` instead.

Upon further reading... you have a comment on the CSS indicating that keeps everyone except the license notification sticky... which immediately suggests to invert that. That is, the default behavior should be sticky; it is the exception where you should have to do something explicit (and thereby making maintenance much less fragile).

Reply:
Having the sticky part contained would be ideal, though not accomplishable in the current state.  

The notification component is only responsible for the component itself.  It makes no distinction as to where it is placed on the page and is only a single ribbon.  On top of that, we want the ribbon to lay over content, and not take up space.  This makes its sticky positioning the responsibility of the container holding it, which sometimes holds multiple notifications at once.

The best way I can think of (for now) to maintain some consistency, is have the sticky container be a global style.  I'll add a note for its use.  In the near future, I hope to refactor our app notifications to use the global notifications container thats already in place and used by about half the notifications. but that will have to be a larger ticket for another time.

I'm thinking my comment wasn't clear enough about the license notification.  The clearer reason I wrote that note, is that i'm actually just _not using_ the sticky class on the license notification container (as opposed to the class not working the same way), so that it doesn't cover any of the top nav   Though UX may change that.

Reply:
double checking and highlighting this led me to a different small issue.  Glad you mentioned it!